src/content/docs/release-notes/docs-release-notes/docs-10-25-2024.mdx

```diff
@@ -0,0 +1,65 @@
+---
+subject: "Docs"
+releaseDate: '2024-10-25'
+version: 'version: October 18-24, 2024'
+---
+
+### New docs
+
+* New troubleshooting doc [PHP agent overhead reduction tips](/docs/apm/agents/php-agent/troubleshooting/php-agent-overhead-reduction-tips) lists configuration settings you can change to reduce the agent's overhead.
```

Comment:
Suggestion: We can drop "New troubleshooting doc "  to maintain consistency with other sections.

[PHP agent overhead reduction tips](/docs/apm/agents/php-agent/troubleshooting/php-agent-overhead-reduction-tips) lists configuration settings you can change to reduce the agent's overhead.

The link seems to be broken